Specifications/Compiler/com.apple.xcode.tools.ibtool.storyboard.compiler.xcspec

```diff
@@ -0,0 +1,43 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+{
+    Type = Compiler;
+    Identifier = "com.apple.xcode.tools.ibtool.storyboard.compiler";
+    Name = "Storyboard Compiler";
+
+    RuleName = "CompileStoryboard $(InputFile)";
+
+    InputFileTypes = (
+        "file.storyboard"
+    );
+
+    CommandLine = "ibtool [options] [special-args] [input]";
+
+    Options = (
+        {
+            Name = "IBTOOL_OUTPUT_FORMAT";
+            Type = String;
+            DefaultValue = "human-readable-text";
+            CommandLineArgs = ( "--output-format", "$(value)" );
```

Comment:
Rather than `CommandLineArgs` with `$(value)`, this can just be `CommandLineFlag = "--output-format";`